ts/features/itwallet/common/store/selectors/index.ts

```diff
@@ -105,3 +105,16 @@ export const itwShouldRenderL3UpgradeBannerSelector = (state: GlobalState) =>
   !offlineAccessReasonSelector(state) &&
   itwIsL3EnabledSelector(state) &&
   itwAuthLevelSelector(state) !== "L3";
+
+/**
+ * Returns whether the new IT-Wallet variant should be rendered.
+ * - The IT Wallet feature flag is enabled
+ * - The wallet is not offline
+ * - The L3 feature flag is enabled
+ * - The wallet is with L3 auth
+ */
+export const itwShouldRenderNewITWallet = (state: GlobalState) =>
```

Comment:
Instead of checking if we have an L3 auth level, wha about checking if the stored PID is an L3 credential?
Checkout this PR suggestion made here: https://github.com/pagopa/io-app/pull/6989#discussion_r2099882040
Do you think it can be a stronger solution?